README.md

```diff
@@ -48,7 +48,17 @@ Run both unit and browser tests:
 
 ### Publish
 
-(*To be done*)
+In order to publish the application on [Github pages](https://pages.github.com/),
+you first need to build and minify the JavaScript files using:
+
+    $ npm run build
+
+Check the content of the `build/` folder, and then use the `gh-pages` utility
+to publish it into the `gh-pages` branch automatically:
+
+    $ ./node_modules/.bin/gh-pages --dist build/
```

Comment:
I wonder if we couldn't provide an `npm run gh-pages-publish` command to smooth all this even further?